cds-snc/cppd-medical-report-sails
# Supporting documents

## Depends on 
- #21 #20 #18

## Adds
- Brings back the Supporting Documents section
- Brings back Vue support
- Fixes File Uploader in Conditions section as well

## To test
- Fill in Personal form
- Add one or more conditions
- Skip Medications
- Skip Treatments
- Submit documents form without uploading anything to test validation
- Check "Attach later" checkbox to check conditional validation
- Will redirect back to Conditions - skip, skip, skip
- Attach some documents and submit
- Will redirect back to Conditions - skip, skip, skip
- See saved documents list
